plugins/woocommerce/templates/loop/pagination.php

```diff
@@ -12,7 +12,7 @@
  *
  * @see     https://woocommerce.com/document/template-structure/
  * @package WooCommerce\Templates
- * @version 3.3.1
+ * @version 9.2.0
```

Comment:
Can you bump this to 9.3.0 as this PR won't get released in 9.2?

Reply:
You can get the information about current release in [Woo Developer Blog](https://developer.woocommerce.com). For example, [this post](https://developer.woocommerce.com/2024/07/26/woocommerce-9-2-pre-release-updates/#h-update-timeline) contains all information related to 9.2 release.

Reply:
@dinhtungdu bumped to 9.3.0